config/routes.rb

```diff
@@ -8,6 +8,6 @@
 
     get "translate", to: "translations#index"
     get "about", to: "static#about"
-    root to: "doctors#search"
+    root to: "doctors#search_root"
```

Comment:
Now I remember why I did this. I cannot figure out how to send JS to be loaded on root page only. Checking for the / path won't work because there can be things added to the path at root. Cannot figure this out.